src/providers/ProjectsProvider.ts

```diff
@@ -6,8 +6,6 @@ import { ProjectEntry } from './entries/ProjectEntry';
 import ProjectsState from '../state/ProjectsState';
 
 export default class ProjectsProvider implements vsc.TreeDataProvider<Entry> {
-  constructor(private ctx: vsc.ExtensionContext) {}
```

Comment:
double checking: this isnt needed anymore?

Reply:
vscode was flagging that this was a unused var